spec/models/project_spec.rb

```diff
@@ -22,6 +22,11 @@
   	expect(Rugged::Repository.new(File.join @project.path , 'satellite' , '.git')).to be_a(Rugged::Repository)
   end
   
+  it "lists public projects for a certain user" do
```

Comment:
Yeah, I'm sorry... The test was different at first then I changed it and forgot to change the name :). I think I should just change the name as the purpose of this test was just to make sure public filtering works.